x-pack/plugins/siem/server/ui_settings.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { schema } from '@kbn/config-schema';
+
+import { CoreSetup } from '../../../../src/core/server';
+import {
+  DEFAULT_INDEX_KEY,
+  DEFAULT_INDEX_PATTERN,
+  DEFAULT_ANOMALY_SCORE,
+  DEFAULT_SIEM_TIME_RANGE,
+  DEFAULT_SIEM_REFRESH_INTERVAL,
+  DEFAULT_INTERVAL_PAUSE,
+  DEFAULT_INTERVAL_VALUE,
+  DEFAULT_FROM,
+  DEFAULT_TO,
+  ENABLE_NEWS_FEED_SETTING,
+  NEWS_FEED_URL_SETTING,
+  NEWS_FEED_URL_SETTING_DEFAULT,
+  IP_REPUTATION_LINKS_SETTING,
+  IP_REPUTATION_LINKS_SETTING_DEFAULT,
+} from '../common/constants';
+
+export const initUiSettings = (uiSettings: CoreSetup['uiSettings']) => {
+  uiSettings.register({
+    [DEFAULT_SIEM_REFRESH_INTERVAL]: {
+      type: 'json',
+      name: i18n.translate('xpack.siem.uiSettings.defaultRefreshIntervalLabel', {
+        defaultMessage: 'Time filter refresh interval',
+      }),
```

Comment:
I think this setting may be broken? Modifying it to the below configuration on either this branch or `master` (siem-dev) does not result in any changes to the datepicker `Refresh every` UI. 

This appears to be pre-existing, so no need to fix here. I've opened this issue to track: https://github.com/elastic/kibana/issues/64034. It looks like the setting isn't being read correctly and it just comes back as `null`.

Config:

``` js
{
  "pause": false,
  "value": 230000
}
```

Result:
<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/79824350-559b8080-8353-11ea-914b-94cb67651a96.png" />
</p>

Reply:
Great catch, I'll follow up on that issue.